rules/linux/lnx_clear_logs.yml

```diff
@@ -0,0 +1,28 @@
+title: Clear Linux Logs
+id: 80915f59-9b56-4616-9de0-fd0dea6c12fe
+status: stable
+description: Detects clear logs
+author: Ömer Günal, oscd.community
+date: 2020/10/07
+references:
+    - https://attack.mitre.org/techniques/T1070/002/
+logsource:
+    product: linux
+    category: process_creation
+detection:
+    keywords:
+        - Commands|contains:
+          - 'rm * /var/log*'
+          - 'shred -u /var/log*'
+          - 'echo * > /var/log*'
+          - 'rmdir * /var/log*'
+          - 'rm * /private/var/audit/*'
+          - 'rm * /private/var/log/system.log*'
+          - 'echo * /var/spool/mail/*'
+    condition: keywords
```

Comment:
Hello @omergunal!
Could you please update your rule using [this rule for macOS](https://github.com/Neo23x0/sigma/pull/1132/files) as an example?